diagnostics/model_accuracy/verify_inference_model.py

```diff
@@ -154,7 +131,11 @@ def check_accuracy(model, p, tolerance=0.001):
     #TODO(Sindhu): Run multiple or ALL models at once and compare accuracy.
 
     try:
-        model_name, p = run_inference(args.model_name, models_dir)
-        check_accuracy(model_name, p)
+        print(args.json_file_name)
```

Comment:
why are we printing json file name here?

Reply:
@sindhu-nervana, there isn't this line "print(args.json_file_name)" there, would you pls double check.

Reply:
@sindhu-nervana , sorry, I saw the line, let me delete it.